README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://travis-ci.org/dbtsai/python-mimeparse.svg?branch=master
+   :target: https://travis-ci.org/dbtsai/python-mimeparse
+
+This module provides basic functions for handling mime-types. It can
+handle matching mime-types against a list of media-ranges. See section
+5.3.2 of the HTTP 1.1 Semantics and Content specification [RFC 7231] for
+a complete explanation: https://tools.ietf.org/html/rfc7231#section-5.3.2
+
+Functions
+=========
+
+``parse_mime_type()``
```

Comment:
You can't use smaller headers in RST without introducing another logical level

I think they will look fine when there is more content though, compare to https://github.com/adamchainz/patchy